backup-stores/s3/src/main/java/io/camunda/zeebe/backup/s3/S3BackupConfig.java

```diff
@@ -103,6 +108,12 @@ public static class Builder {
     private Credentials credentials;
     private String basePath;
 
+    /** Default from `SdkHttpConfigurationOption.MAX_CONNECTIONS` */
+    private Integer maxConcurrentConnections = 50;
+
+    /** Default from `SdkHttpConfigurationOption.DEFAULT_CONNECTION_ACQUIRE_TIMEOUT` */
+    private Duration connectionAcquisitionTimeout = Duration.ofSeconds(10);
```

Comment:
:x: Unless I'm missing something (it's friday after all :sweat_smile:) this changes the default timeout from 45 seconds to 10 seconds. I don't think we should do that.